README.md

```diff
@@ -1,26 +1,56 @@
 
-# About
+# lsif-util
 
-Scripts to help [LSIF](https://github.com/Microsoft/language-server-protocol/blob/master/indexFormat/specification.md) developers. Featuring:
+Scripts to help [LSIF](https://github.com/Microsoft/language-server-protocol/blob/master/indexFormat/specification.md) developers.
```

Comment:
It would be useful to point out the LSIF spec that is supported by the tool

Reply:
Now that you say that, I went to check the documentation and the latest version there is 0.2.2. However, we are using version 0.3.1 of the protocol. Am I completely wrong and these numbers are unrelated? Or is the documentation lagging behind?

Reply:
Yes the docs are lagging behind. The version of the lsif-protocol npm package is accurate. I’ll make a PR to fix the docs

Reply:
I will keep this request in my heart but I will hold back for now. I want to eventually set up continuous integration for this repo (since I now have some handy unit tests). I will wait until the code migrated to lsif-node though, otherwise I would have to set up everything twice. At this point, we will be "in sync" with the protocol package, so I believe it will be more convenient to display extra info.